workload/tests/check_client.cc

```diff
@@ -7,11 +7,11 @@
  * Copyright (c) 2020 CESAR
  */
 
-#include "svid/x509svid/src/svid.h"
-#include "workload/src/client.h"
 #include "../workload.grpc.pb.h"
 #include "../workload.pb.h"
 #include "../workload_mock.grpc.pb.h"
```

Comment:
I know this is not part of PR, but didn't we agree in not using `../` includes?

Reply:
fixed on commit [7b012c2bfe2d0011efd0ff3263cab24505c8384c](https://github.com/HewlettPackard/c-spiffe/pull/53/commits/7b012c2bfe2d0011efd0ff3263cab24505c8384c)